tests/app/celery/test_process_ses_receipts_tasks.py

```diff
@@ -38,16 +38,6 @@ def test_process_ses_results_retry_called(sample_email_template, mocker):
     assert mocked.call_count != 0
 
 
-def test_process_ses_results_in_complaint(sample_email_template, mocker):
```

Comment:
I couldn't see another test for this scenario of a standard SES callback creating a complaint, so perhaps we'd want to just rewrite this instead of deleting it completely? The `process_ses_results` doesn't call `update_notification_status_by_reference`, so the check that that function isn't being called might be left over from some old code anyway

Reply:
Good spot! I've tweaked that commit to keep this test in (and assert the notification isn't updated at all)